cpp/include/mcts/UniformNNEvaluationService.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <core/concepts/Game.hpp>
+#include <mcts/NNEvaluation.hpp>
+#include <mcts/NNEvaluationRequest.hpp>
+#include <mcts/NNEvaluationServiceBase.hpp>
+
```

Comment:
I would add a comment above this class offering a description of how this service behaves, and why this class exists (to support gen-0 self-play, when we don't yet have a nn model).